app/src/main/java/com/example/ubclaunchpad/launchnote/edit/PhotoInfoActivity.kt

```diff
@@ -95,9 +95,54 @@ class PhotoInfoActivity : AppCompatActivity() {
         picNoteToEdit.title = title_input.text.toString()
         picNoteToEdit.description = description_input.text.toString()
         // todo vpineda we need to create a dropdown for all of the classes or projects!
-        if(folder_input.text.isNotEmpty())
+        if(folder_input.text.isNotEmpty()) {
             picNoteToEdit.folderId = Integer.parseInt(folder_input.text.toString())
 
+            var folderToInsert = Folder()
+
+            // create new Folder
+            LaunchNoteDatabase.getDatabase(this)?.let {
```

Comment:
Can we move this function to a blocking function or to a function that receives a callback. We might need this for other things such as the info panel (probably)